doc/tutorials/using_windows_as_uos.rst

```diff
@@ -313,90 +313,83 @@ Run these commands on the Service VM::
 
 Install Windows 10
 ------------------
-Currently, the ACRNGT OVMF GOP driver is not ready; thus, a special VGA
-version is used to install Windows 10 on ACRN from scratch. The
-``acrn.efi``, ``acrn-dm`` and ``OVMF`` binaries are included in the
-`tarball
-<https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/tutorials/install_by_vga_gsg.tar.gz>`_
-together with the script used to install Windows 10.
 
-#. Uncompress ``install_by_vga_gsg.tar.gz`` to the Service VM::
+   .. note:: Please make sure you have configured your monitor and display according to **3** of
+      :ref:`Boot Windows with GVT-g on ACRN <waag_display_conf_lable>`
 
-   # tar zxvf install_by_vga_gsg.tar.gz && cd install_by_vga_gsg
+#. Make a copy from ``/usr/share/acrn/samples/nuc/launch_win.sh`` to ``install_win.sh``::
 
-#. Edit the ``acrn-dm`` command line in ``install_vga.sh`` if your configuration is different.
+   # cp /usr/share/acrn/samples/nuc/launch_win.sh ~/install_win.sh
 
-   - Change ``-s 3,virtio-blk,./win10-ltsc-virtio.img`` to your path to the Windows 10 image.
-   - Change ``-s 8,ahci,cd:./windows10-17763-107-LTSC-Virtio-Gfx.iso`` to the ISO you re-generated above.
-   - Change ``-s 9,ahci,cd:./virtio-win-0.1.141.iso`` to your path to the virtio-win iso.
-   Or used Oracle driver
-   - Change ``-s 9,ahci,cd:./winvirtio.iso`` to your path to the winvirtio iso.
+#. Add the following lines **before** ``acrn-dm`` command line in ``install_win.sh``. It is used
+   to passthrough USB to WaaG, by which both mouse and keyboard can be used during Windows installation::
+
+      echo "8086 9d2f" > /sys/bus/pci/drivers/pci-stub/new_id
+      echo "0000:00:14.0" > /sys/bus/pci/devices/0000:00:14.0/driver/unbind
+      echo "0000:00:14.0" > /sys/bus/pci/drivers/pci-stub/bind
+
+   .. note:: You may need to change the bdf and vid/pid of the USB controller in above command.
+      You can use ``lspci`` and ``lspci -n`` to get this information.
 
-#. Run ``install_vga.sh`` and connect to the Windows guest using a vnc client.::
+#. Edit the ``acrn-dm`` command line in ``install_win.sh`` as follows:
+
+   - Change ``-s 3,virtio-blk,./win10-ltsc-virtio.img`` to your path to the Windows 10 image.
+   - Add ``-s 6,passthru,0/14/0``.
```

Comment:
Does that need to be adjusted in case the BDF/VID/PID is different?

Reply:
fix in new PR. Please check